bin/ddg2excel

```diff
@@ -26,12 +26,14 @@ from mutatex.utils import *
 
 LOGFMT = "%(levelname)s: %(message)s"
 
-description = 'ddg2excel: write the average values from a MutateX DDG scan as Excel tables'
-epilog = """ddg2excel writes the average values from a MutateX DDG scan as an Excel table
-file (.xlsx). Option -T swaps rows and columns of the output file. By default,
-residue type, number and chain identifier are written in separate columns/rows
-to make the data easily explorable, however one column/row is used when the
-label file is provided (option -b)"""
+description = 'ddg2tabel: writes the average values from a MutateX DDG scan as tables either in excel or in a csv file format'
+epilog = """ddg2tabel writes the average values from a MutateX DDG scan as a 
```

Comment:
If I remember correctly, Matteo and Elena didn't want to change the tool name righT

Reply:
Corrected. Thank you!